modules/apps/frontend-js/frontend-js-test/src/testFunctional/macros/ManageTranslations.macro

```diff
@@ -10,8 +10,6 @@ definition {
 
 	macro deleteLanguage {
 		Click(locator1 = "ManageTranslations#LIST_ITEM_DELETE_LANGUAGE_BUTTON");
-
-		Click(locator1 = "Button#DONE");
 	}
```

Comment:
Did the behavior of Translation Manager change and there's no longer a `Done` button?